src/test/java/BeforeAfter.java

```diff
@@ -0,0 +1,64 @@
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Execution(ExecutionMode.CONCURRENT)
+public class BeforeAfter {
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    private static final String URL_TO_HUB = "http://localhost:4444/wd/hub";
+
+    public static WebDriver getDriver(String browser, String runMode) throws MalformedURLException {
+        switch (runMode) {
+            case "local":
+                if (browser.equals("chrome")) {
+                    return new ChromeDriver();
+                }
+                return new FirefoxDriver();
+            case "grid":
+                return new RemoteWebDriver(new URL(URL_TO_HUB), getBrowserCapabilities(browser));
+        }
+        return null;
```

Comment:
Return null is always dangerous, because you will get NPE if runMode is not equal to grid/local. I recommend to create your own exception extended from RuntimeException and throw it here.